android/build.gradle

```diff
@@ -174,22 +321,6 @@ def CMAKE_NODE_MODULES_DIR = CLIENT_SIDE_BUILD
         ? project.getProjectDir().getParentFile().getParent()
         : Paths.get(project.getProjectDir().getParent(), getPlaygroundAppName(), "node_modules").toString()
 
-def JS_RUNTIME = {
-    // Returns the js runtime explicitly requested in the environment variable
-    if (System.getenv("JS_RUNTIME")) {
-        return System.getenv("JS_RUNTIME")
-    }
-
-    // Detect js runtime from project setup
-    def defaultRuntimeType = "jsc";
-    def v8Project = rootProject.getSubprojects().find { project -> project.name == "react-native-v8" }
-    if (v8Project != null) {
-        return "v8"
-    }
-    def appProject = findProject(":app") ? project(":app") : null
-    return appProject?.ext?.react?.enableHermes ? "hermes" : defaultRuntimeType
-}.call()
```

Comment:
Havent tested the behaviour, but code wise it looks fine. 

One note: do we want to remove support for v8?

Reply:
Also these changes: https://github.com/software-mansion/react-native-reanimated/pull/3475

Reply:
And these: https://github.com/software-mansion/react-native-reanimated/pull/3566